src/model/Movement.hs

```diff
@@ -9,18 +9,43 @@ import Model.Parameters (screenMinX, screenMaxX, screenMinY, screenMaxY)
 
 type Position = (Float, Float)
 type Vector   = (Float, Float) -- x and y component
+data Square = Square {bottomLeft :: Position, width :: Float} -- A square is defined by the position of the bottom left corner and the width/height.
 
 move :: Position -> Vector -> Position
 move (x, y) (dx, dy) = (x + dx, y + dy)
 
 class HasPosition a where
   pos :: a -> Position
 
+  -- Given an object that has a position and a hitBoxSize, return its hitbox
+  hitbox :: a  -> Square
+  hitbox a = Square {bottomLeft = squarePos, width = hitboxSize}
+    where 
+      hitboxSize = 10
```

Comment:
Het lijkt me beter om dit een functie van de class te maken, en dan voor elke instance de grootte van de Picture die je gebruikt voor de view te returnen. Dan past de hitbox ook echt bij elk object.

Reply:
Oh en de bullets zijn rechthoeken maar geen vierkanten op dit moment, dus als we het heel netjes willen doen moeten we of dat aanpassen of de hitboxes rechthoeken maken.